opta/commands/apply.py

```diff
@@ -78,6 +74,9 @@
     default=False,
     help="Show full terraform plan in detail, not the opta provided summary",
 )
+@config_option
+@env_option
+@input_variable_option
```

Comment:
I wonder if we should also support using a env file to set these variables, similar to `-var-file` in terraform.

we already have a similar option when we set secrets
https://github.com/run-x/opta/blob/main/opta/commands/secret.py#L25
https://github.com/run-x/opta/blob/main/opta%2Fcore%2Fsecrets.py#L39

you could make an extra option  `@input_variable_file_option` and have an extra arg to `InputVariable.parse_string_to_dict(var, var_file)`

definitely not a must have for now but if you think this is useful, you can take a closer look

Reply:
for later lol